### DISCUSSION TITLE
ETA on session Record on Android with WebXR Backend?

### DISCUSSION THREAD

**Discussion by Podden** [google/model-viewer#2110] [General]
Is there any ETA on this? I've read in the issues that you're activly working on it.
We want to expand our Model with more interaction, AR Quicklook lets us use buttons for playing animations and sound and a lot more, scene-viewer on the other hand is very restrictive, but the WebXR Backend lacks the "Take a Photo" features, our customers love so much ;).

**Comment by elalish** [comment]
We're waiting on Chrome for this feature, so it'll be some months yet.
